elasticjob-error-handler/elasticjob-error-handler-type/elasticjob-error-handler-dingtalk/pom.xml

```diff
@@ -33,6 +33,11 @@
             <artifactId>elasticjob-error-handler-spi</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere.elasticjob</groupId>
+            <artifactId>elasticjob-error-handler-general</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
```

Comment:
Is it necessary to introduce general into dingtalk module?

Reply:
JobErrorHandlerFactory is needed because it is in the general module